torch/csrc/jit/passes/quantization.cpp

```diff
@@ -11,6 +11,20 @@ namespace torch {
 namespace jit {
 namespace {
 // QuantizerUtils
+enum insert_position {
+  BEFORE_INSERTION_POINT=0, //Default behaviour
+  AFTER_INSERTION_POINT,
+};
+
+void insertNodeHelper(Node* new_n, Node* ins_node, insert_position pos) {
+  AT_ASSERT(ins_node != nullptr);
+  AT_ASSERT(new_n != nullptr);
+  if (pos == AFTER_INSERTION_POINT) {
+    new_n->insertAfter(ins_node);
+  } else {
+    new_n->insertBefore(ins_node);
+  }
+}
 
 bool checkIfNodeQuantizable(Node* n) {
   AT_ASSERT(n != nullptr);
```

Comment:
We will need to look for op patterns  (conv->relu), here instead of individual ops. Please have a TODO comment and address in a subsequent PR.